src/api/v4/layer/layer.js

```diff
@@ -25,14 +36,18 @@ var metadataParser = require('./metadata/parser');
  * @fires carto.layer.styleChanged
  * @fires carto.layer.MetadataEvent
  * @example
- * // no options
+ * // Create a layer with no options
  * new carto.layer.Layer(citiesSource, citiesStyle);
  * @example
- * // with options
+ * // Create a layer indicating which columns in the data will throw events on mouse over
```

Comment:
Create a layer indicating what columns will be included in the `featureOver` event.